src/go/rpk/pkg/cli/cmd/redpanda_darwin.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Vectorized, Inc.
```

Comment:
Nit, please update Copyright to: Copyright 2022 Redpanda Data, Inc

Reply:
Fixed!